### PR TITLE
Generating Aether models using latest config-model compiler

### PR DESCRIPTION
**Pull request by teone**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ SHELL 			  		= bash -e -o pipefail
 KIND_CLUSTER_NAME 		?= kind
 DOCKER_USER       		?=
 DOCKER_PASSWORD   		?=
-MODEL_COMPILER_VERSION  ?= v0.9.15
+MODEL_COMPILER_VERSION  ?= v0.9.16
```
Comment (SeanCondon) [inline Makefile:9]: should be `0.9.17` to get the latest

File: models/aether-2.0.x/go.mod [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+module github.com/onosproject/aether-models/models/aether-2.1.x
 
 go 1.16
 
 require (
-	github.com/SeanCondon/xpath v0.0.0-20220221181849-c19b784b9842
+	github.com/SeanCondon/xpath v0.0.0-20220225125014-419de017925c
 	github.com/ghodss/yaml v1.0.0
-	github.com/onosproject/config-models v0.0.0-20220224180004-32567adb595a
+	github.com/onosproject/config-models v0.9.15
```
Comment (SeanCondon) [inline models/aether-2.0.x/go.mod:8]: should be `v0.9.17`

**Review by SeanCondon — COMMENTED** [review]
@teone I thought we'd create a whole new model folder for `v2.1.x` - instead of just updating it here. The `v2.0.x` model line will live on in Aether 2.1, so I don't think is should be relegated to a branch

**Comment by SeanCondon** [comment]
@teone can i propose https://github.com/onosproject/aether-models/pull/26 instead - where I create a new `aether-2.1.x` tree altogether

**Comment by teone** [comment]
Once https://github.com/onosproject/aether-models/pull/33 this will be obsoleted

**Pull request closed without merging**